strr-examiner-web/app/components/ApplicationInfoHeader.vue

```diff
@@ -44,20 +44,32 @@ const nocCountdown = computed(() => {
 <template>
   <div class="border-b bg-white py-6">
     <div class="app-inner-container">
-      <div class="mb-2 text-2xl">
-        <strong>
-          {{ activeHeader?.applicationNumber }} |
-        </strong>
-        {{ getApplicationName() }}
+      <div class="flex justify-between mb-2 text-2xl">
```

Comment:
Before committing, you can run `pnpm lint --fix` to fix these link warnings.

Reply:
Fixed